uiowa/uiowa
# Added start homepage feat books view

Closes #4661 


# How to test


1. checkout branch
2. blt ds --site=uipress.uiowa.edu
3. drush @uipress.local cim
4. drush @uipress.local cr
5. drush @uipress.local uli
6. add a couple books with a feat img
7. go to a page, add 'featured books' list (view) block
8. make sure shows up/displays
